server/GameOn/GameOn.Tournaments/Calculators/EloScoreCalculator.cs

```diff
@@ -10,7 +10,29 @@ public class EloScoreCalculator : IScoreCalculator
     {
         public ScoreResult[] Calculate(ScoreResult[] scores, string winnerId)
         {
-            throw new NotImplementedException();
+            ScoreResult playerOne = scores[0];
+            ScoreResult playerTwo = scores[1];
+
+            int outcome = 1;
+
+            if (winnerId == playerOne.PlayerId)
+                outcome = 0;
+
+            int eloK = 32;
+
+            int delta = (int)(eloK * (outcome - ExpectationToWin(playerOne.Score, playerTwo.Score)));
```

Comment:
Be aware that casing to int will not round - is that OK? If you want to round, use `Math.Round()`